docs/incubating/identitycenter_custom_policy_attachment.md

```diff
@@ -0,0 +1,50 @@
+---
+subcategory: "IAM Identity Center"
+---
+
+# huaweicloud_identitycenter_custom_policy_attachment
+
+Manages an Identity Center custom policy attachment resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "permission_set_id" {}
+
+data "huaweicloud_identitycenter_instance" "system" {}
+
+resource "huaweicloud_identitycenter_custom_policy_attachment" "test" {
+  instance_id       = data.huaweicloud_identitycenter_instance.system.id
+  permission_set_id = var.permission_set_id
+  custom_policy     = "{\"Version\":\"5.0\",\"Statement\":[{\"Effect\":\"Allow\",\"Action\":[\"eps:resources:add\"]}]}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `instance_id` - (Required, String, ForceNew) Specifies the ID of the IAM Identity Center instance.
+
+  Changing this parameter will create a new resource.
+
+* `permission_set_id` - (Required, String, ForceNew) Specifies the ID of the IAM Identity Center permission set.
+
+  Changing this parameter will create a new resource.
+
+* `custom_policy` - (Required, String) Specifies the custom policy to attach to a permission set.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID which is formatted `<instance_id>/<permission_set_id>`.
```

Comment:
the format is permission_set_id in the code

Reply:
fixed